core/src/main/scala/kafka/coordinator/group/GroupMetadataManager.scala

```diff
@@ -283,36 +284,7 @@ class GroupMetadataManager(brokerId: Int,
           } else {
             debug(s"Metadata from group ${group.groupId} with generation $generationId failed when appending to log " +
               s"due to ${status.error.exceptionName}")
-
-            // transform the log append error code to the corresponding the commit status error code
-            status.error match {
-              case Errors.UNKNOWN_TOPIC_OR_PARTITION
-                   | Errors.NOT_ENOUGH_REPLICAS
-                   | Errors.NOT_ENOUGH_REPLICAS_AFTER_APPEND =>
-                Errors.COORDINATOR_NOT_AVAILABLE
-
-              case Errors.NOT_LEADER_OR_FOLLOWER
-                   | Errors.KAFKA_STORAGE_ERROR =>
-                Errors.NOT_COORDINATOR
-
-              case Errors.REQUEST_TIMED_OUT =>
-                Errors.REBALANCE_IN_PROGRESS
-
-              case Errors.MESSAGE_TOO_LARGE
-                   | Errors.RECORD_LIST_TOO_LARGE
-                   | Errors.INVALID_FETCH_SIZE =>
-
-                error(s"Appending metadata message for group ${group.groupId} generation $generationId failed due to " +
```

Comment:
We lost the error messages. We could probably keep them in the new utility and just pass through the groupId (i.e. leave out the generation). Probably overkill to do some kind of custom callback.

Reply:
I pulled from the wrong method 🤦‍♀️ the other one doesn't have the message.